src/components/button/Button.js

```diff
@@ -0,0 +1,14 @@
+import './button.css'
+
+function Button(props) {
+  var buttonText = props.buttonText
```

Comment:
@Eduardclt01 you don't need to assign the prop to a local variable, unless you're doing using "use default value if undefined" type of strategy, e.g.

`var buttonText = props.buttonText || 'Default Button Label';

In this case you can just use the props directly because you're not catching the value if it's undefined, so just:
```
import './button.css'

function Button(props) {
  return (
    <button onClick={props.onButtonClick} className='button'>
      {props.buttonText}
    </button>
  );
}

export default Button;
```
would work